python/cuml/dask/common/utils.py

```diff
@@ -178,6 +179,11 @@ def serialize_mat_descriptor(m):
         register_generic(MultinomialNB, "dask",
                          dask_serialize, dask_deserialize)
 
+        register_generic(UMAP, "cuda",
```

Comment:
Since UMAP extends `cuml.Base`, we should no longer need this to be explicitly registered, right? Actually, I think that's the same for MultinomialNB. I'll have to check that one.

Reply:
Yes